.vaunt/config.yaml

```diff
@@ -0,0 +1,89 @@
+version: 0.0.1
+achievements:
+    - achievement:
+          name: Docs Dash PR Submitted
+          icon: <icon_url>
+          description: Submit a PR with the label 2024-tidb-docs-dash between January 9th - 12th 2024!
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition: "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & labels in {'2024-tidb-docs-dash','contribution'}"
```

Comment:
Just to confirm, `labels in {'2024-tidb-docs-dash','contribution'}` means a PR with either of the two labels?

Reply:
This condition `labels in {'2024-tidb-docs-dash','contribution'}` actually means that the PR must have both of these labels to count.  If you want to support having either of the two labels you need to use square brackets, i.e.: `labels in ['2024-tidb-docs-dash','contribution']`